requirements/base.txt

```diff
@@ -60,7 +60,7 @@ cron-descriptor==1.2.24
     # via apache-superset
 croniter==1.0.15
     # via apache-superset
-cryptography==3.4.7
+cryptography==39.0.0
```

Comment:
strange change, prob joined the major and minor https://pypi.org/project/cryptography/#history